### DISCUSSION TITLE
feature: switch audio devices when user switches them

### DISCUSSION THREAD

**Discussion by Cheshire92** [AscensionGameDev/Intersect-Engine#2709] [Ideas]
**Is your feature request related to a problem? Please describe.**
When a user changes their audio output device the audio keeps playing on the old device until the client is restarted. 

**Describe the solution you'd like**
In most modern games and such the audio switches over to the device with it. Might be worth looking into as a minor enhancement at some point.

**Comment by pandinocoder** [comment]
This requires significant refactoring in MonoGame or a complete re-implementation of the sound system, seems minor, unfortunately is not (I would have fixed it by now otherwise since I'm personally affected by this).

**Comment by pandinocoder** [comment]
This is effectively both a bug and a feature request.

**Comment by pandinocoder** [comment]
Changing this to just a feature request because it doesn't make sense to track such a low-priority high-difficulty change as a bug.
